mautic/mautic
# Add a new stage condition in the campaign

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



Currently, the user cannot add a condition based on the contacts stage in the campaign. So, if you want to use this condition, you will need to create a segment for each stage. 

This PR adds this condition
![image](https://user-images.githubusercontent.com/96085911/172209438-b9c3a456-14d8-4d29-bdfa-70445c28b1ab.png)


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new campaign
3. Try to add a new condition
4. A contact stage condition here
5. Add this condition and build a campaign
6. The condition works when the campaign is triggered




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11222"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
